### PR TITLE
[test] Add IDL test for Web ML Polyfill API.

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin PTAL, thanks.

### CODE REVIEW

File: test/wpt/interfaces/webml.idl [hunk 1]
```diff
@@ -0,0 +1,93 @@
+partial interface Navigator {
```
Comment (huningxin) [inline test/wpt/interfaces/webml.idl]: we'd better to keep one version inside `doc` folder, can we?

File: test/wpt/webml/idlharness.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Web ML Polyfill API IDL Tests</title>
+<script src="../resources/testharness.js"></script>
```
Comment (huningxin) [inline test/wpt/webml/idlharness.html]: What's the license of files inside `resources`? If they are from third_party, we'd better to put a README.md to explain.

**Comment by BruceDai** [comment]
@halton @huningxin PTAL, thanks.

**Comment by huningxin** [comment]
Looks good. Thanks!

**Pull request merged**